services/base.js

```diff
@@ -318,28 +327,26 @@ class BaseService {
   }
 
   static register({ camp, handleRequest, githubApiProvider }, serviceConfig) {
-    const ServiceClass = this // In a static context, "this" is the class.
```

Comment:
Some references use `this` and some use `ServiceClass`. Don't really care which we use, though consistent = better.